Src/IronPython/Runtime/Operations/IntOps.cs

```diff
@@ -551,6 +551,30 @@ public static partial class Int32Ops {
             }
         }
 
+        public static int __round__(int number) {
+            return number;
+        }
+
+        public static object __round__(int number, BigInteger ndigits) {
+            var result = BigIntegerOps.__round__(new BigInteger(number), ndigits);
+            if (result.AsInt32(out var ret)) {
+                return ret;
+            }
+
+            // this path can be hit when number is close to int.MaxValue and ndigits is negative,
+            // causing number to be rounded up and over int.MaxValue
+            return result;
+        }
+
+        public static object __round__(int number, int ndigits) {
```

Comment:
Probably don't need this overload if all it does it cast `ndigits` to bigint.

Reply:
agreed